File: C#/6/6_2.cs
void ProcessGrade(int grade)
{
    if (grade >= 90)
        HandleGrade("A", true);
    else if (grade >= 80)
        HandleGrade("B", true);
    else if (grade >= 70)
        HandleGrade("C", false);
    else if (grade >= 60)
        HandleGrade("D", false);
    else
        HandleGrade("F", false);
}

void HandleGrade(string gradeLetter, bool shouldCongratulate)
{
    Console.WriteLine("Grade: " + gradeLetter);

    if (shouldCongratulate)
    {
        CongratulateStudent();
    }

    UpdateRecords();
}

File: Python/6/6_2.py
if grade >= 90:
    print("Grade: A")
    congratulate_student()
    update_records()
elif grade >= 80:
    print("Grade: B")
    congratulate_student()
    update_records()
elif grade >= 70:
    print("Grade: C")
    update_records()
elif grade >= 60:
    print("Grade: D")
    update_records()
else:
    print("Grade: F")
    update_records()
